lib/runtime.js

```diff
@@ -125,13 +112,37 @@ exports.enable = function (Module) {
     if (name === "default" &&
         ! (exports &&
            typeof exports === "object" &&
-           exports.__esModule &&
+           hasOwn.call(exports, __esModule) &&
            "default" in exports)) {
       return exports;
     }
 
-    return exports && exports[name];
+    if (exports == null) {
+      return;
+    }
+
+    return exports[name];
   };
 
   return Module;
 };
+
+function setESModule(module) {
+  var exports = module.exports;
+  if (exports &&
+      typeof exports === "object" &&
+      ! hasOwn.call(exports, __esModule)) {
+
+    if (typeof __esModule === "string") {
+      Object.defineProperty(exports, __esModule, {
+        value: true,
+        enumerable: false,
+        writable: false,
+        configurable: true
+      });
+    } else {
+      exports[__esModule] = true;
+    }
```

Comment:
Here is the toggle for symbol or string use. If it's a string then a non-enumerable property is set otherwise a symbol (which is non-enumerable in the traditional sense) is used.